ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,71 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: docker-container-executor/Dockerfile
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: rhsso-runner
+releases:
+  latest:
+    prerelease:
+      product: ocp
+      version_bounds:
+        lower: 4.13.0-0
+        stream: 4-stable
+        upper: 4.14.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhsso-interop-aws
+  cron: 0 6 25 12 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI",  "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+              {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "keycloak-qe-rhsso-tests", "failure_type": "pod_failure", "classification": "Test Execution", "jira_project": "LPTOCPCI",  "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "keycloak-qe-rhsso-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "RHSSO", "group": {"name": "lp-tests", "priority": 2},"jira_additional_labels": ["rhsso","!default","interop-tests"],"jira_assignee":"ikhomyn@redhat.com"}
+            ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.14-lp","self-managed-lp","rhsso-lp","z-stream"]'
```

Comment:
on (12/21/23) in our canvas meeting notes we discussed the need for a unique identifier.

I don't believe we captured all resolutions there but to recap I think we agreed that unuqiue identifiers for zstream should be avoided and that we will need to rely on knowledge of the current OCP pre-release version to understand which versions are our zstream versions.

This philopsophy would lead to removing the label `z-stream` here as we would know that we need to filter on 4.14 as our N-1 version. Our Jira filters could be zstream specific if the watchers needed it, essentially search for all 4.14 and call it filter z-stream N-1. 

Removing z-stream here helps us when all of the current pre-release 4.15 jobs become the zstream n-1 version. If we add this label to z-stream now then when 4.15 becomes zstream we will need to add this label to every file manually.

Reply:
Yes I agree, it makes sense to avoid adding the label for z-stream here